specification/edgeorder/resource-manager/Microsoft.EdgeOrder/2020-12-01-preview/examples/OrderCollectionCreate.json

```diff
@@ -0,0 +1,57 @@
+{
+  "parameters": {
+    "subscriptionId": "fa68082f-8ff7-4a25-95c7-ce9da541242f",
+    "resourceGroupName": "TestRG",
+    "orderCollectionName": "TestOrderCollectionName1",
+    "api-version": "2020-12-01-preview",
+    "User-Agent": [
+      "vscode-restclient"
+    ],
+    "Referer": [
+      "https://localhost/subscriptions/fa68082f-8ff7-4a25-95c7-ce9da541242f/resourceGroups/TestRG/providers/Microsoft.EdgeOrder/orderCollections/TestOrderCollectionName1?api-version=2020-12-01-preview"
+    ],
+    "Content-Type": [
+      "application/json; charset=utf-8"
+    ],
+    "Content-Length": [
+      "447"
+    ],
+    "orderCollectionResource": {
+      "properties": {
+        "OrderIds": [
+          "/subscriptions/fa68082f-8ff7-4a25-95c7-ce9da541242f/resourcegroups/TestRG/providers/Microsoft.EdgeOrder/Orders/TestOrderName1",
+          "/subscriptions/fa68082f-8ff7-4a25-95c7-ce9da541242f/resourcegroups/TestRG1/providers/Microsoft.EdgeOrder/Orders/TestOrderName2"
```

Comment:
so cross RG orders are allowed? you should probably have a linkedAccess check to ensure that the user creating the order collection has permission to read the order(s) specified in the request body

Reply:
Yes cross RG orders are allowed. Sure